modules/client/examples/03-tokenVoting-client/11-get-members.ts

```diff
@@ -30,11 +34,25 @@ Returns:
 
 ```json
 [
-  "0x1234567890123456789012345678901234567890",
-  "0x2345678901234567890123456789012345678901",
-  "0x3456789012345678901234567890123456789012",
-  "0x4567890123456789012345678901234567890123",
-  "0x5678901234567890123456789012345678901234"
+  {
+    "address": "0x1234567890123456789012345678901234567890",
+    "balance": 100n,
```

Comment:
Will these values be always up to date? Can subgraph monitor all this kind of token transfers?

Reply:
I think so, since the token used for voting is a `GovernanceErc20` deployed by us i believe all transfers are monitored, @Rekard0  can you confirm?